src/get-started/labels-legends-headings/address-group/address-wrapper.scss

```diff
@@ -0,0 +1,11 @@
+@import "example-init";
```

Comment:
This doesn't seem to be used by the example - the example uses the grid and doesn't import this using the `stylesheets` frontmatter.